bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
+    locale = l10n_utils.get_locale(request)
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
+    if lang_file_is_active(new_home, locale):
```

Comment:
Besides the first param that should be `firefox/os/index-new`, it should be correct to use `lang_file_is_active` per [bug discussion](https://bugzilla.mozilla.org/show_bug.cgi?id=1125898#c30).